static/js/components/PrivacyForm.js

```diff
@@ -12,17 +13,24 @@ import type {
 } from "../flow/profileTypes"
 import type { UIState } from "../reducers/ui"
 import type { Validator } from "../lib/validation/profile"
+import { sendFormFieldEvent } from "../lib/google_analytics"
+import type { Option } from "../flow/generalTypes"
```

Comment:
Can you move this and the other `import type` lines to a group below the other imports?